src/styles/foundation/_focus-ring.scss

```diff
@@ -1,18 +1,17 @@
 /// Sets the focus ring for an interactive element
 /// @param {String} $size - The size of the border radius on the focus ring.
 /// @param {String} $style - Focus ring state.
-/// @param {Number} $border-width - The border width of your element in rems.
+/// @param {Number} $border-width - The border width of your element in px.
 ///
 
-@mixin focus-ring($size: 'base', $border-width: rem(0), $style: 'base') {
+@mixin focus-ring($size: 'base', $border-width: '0px', $style: 'base') {
```

Comment:
I had to change this because `rem(0)` outputs `0` without `px` and that will break the border-radius calc below, resulting in a 0px border radius if `$border-width` is not passed to the mixin

i.e. 
`calc(var(--p-border-radius-base) + 0 + 0.1rem)` will render no border radius
vs
`calc(var(--p-border-radius-base) + 0px + 0.1rem)` will render a proper border radius

Reply:
I agree. We're after 0px in the end and 0 is 0.